src/Controller/UserController.php

```diff
@@ -14,4 +19,45 @@ public function login(): void
     {
         $this->render('layout/login.html.twig');
     }
+
+    /**
+     * @throws ReflectionException
+     * @throws Exception
+     */
+    public function signup(): void
+    {
+        if (empty($_POST)) {
```

Comment:
Idem than AdminController only one render and test if form is submit and valid condition